testing/fixtures/fixtures.go

```diff
@@ -12,10 +12,11 @@ import (
 
 // Project describes a git repo for a test fixture and env vars for when it is to be built
 type Project struct {
-	Name   string
-	URL    string
-	Commit string
-	Env    map[string]string
+	Name    string
```

Comment:
[off-topic] It is super annoying that realignment from `go fmt` causes such noisy diffs.